plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -2,6 +2,67 @@
 <?eclipse version="3.4"?>
 <plugin>
 
+  <extension point="org.eclipse.ui.commands">
+    <command
+      description="%deployStandardCommandDescription"
+      id="com.google.cloud.tools.eclipse.appengine.deploy"
+      name="%deployStandardCommandName">
+    </command>
+  </extension>
+
+  <extension point="org.eclipse.ui.menus">
+    <menuContribution
+      allPopups="false"
+      locationURI="popup:org.eclipse.ui.popup.any?after=additions">
+      <command
+        commandId="com.google.cloud.tools.eclipse.appengine.deploy"
+        label="%deployStandardMenuLabel"
+        style="push">
+        <visibleWhen checkEnabled="true" />
+      </command>
+    </menuContribution>
+  </extension>
+
+  <extension point="org.eclipse.ui.handlers">
+    <handler
+      class="com.google.cloud.tools.eclipse.appengine.deploy.ui.standard.StandardDeployCommandHandler"
+      commandId="com.google.cloud.tools.eclipse.appengine.deploy">
+      <enabledWhen>
+        <and>
+          <count value="1" />
+          <iterate>
+            <reference definitionId="com.google.cloud.tools.eclipse.appengine.onlyInGCPProjects" />
+          </iterate>
+        </and>
+      </enabledWhen>
+    </handler>
+  </extension>
+
+  <extension
+    point="org.eclipse.ui.console.consolePatternMatchListeners">
+    <consolePatternMatchListener
+      class="com.google.cloud.tools.eclipse.ui.util.console.PatternToHyperlinkConverter"
+      id="com.google.cloud.tools.eclipse.appengine.deploy.consolePatternMatchListener"
+      regex="https://.+\.appspot\.com">
```

Comment:
I missed this before, but we should constrain this regex as `.+` is unnecessarily broad.  `[a-zA-Z][a-zA-Z0-9-.]+[a-zA-Z0-9]`?  (I'm not sure about the `.`: I know AE is moving from _version.service.app_.appspot.com to the _version_-_service_-_app_.appspot.com model, but does that show here?)